ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml

```diff
@@ -74,6 +74,27 @@ tests:
     - chain: ipi-aws-pre
     - chain: workers-scale
     workflow: openshift-qe-node-density-heavy
+- as: ocp-qe-perfscale-ingress-perf-3nodes-periodic-aws
+  cron: 0 9 * * 3
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "9"
+      COMPUTE_NODE_TYPE: m5.2xlarge
+      CONFIG: config/standard-classic-rosa.yml
+      CONTROL_PLANE_INSTANCE_TYPE: r5.xlarge
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: c5.4xlarge
+      OPENSHIFT_INFRA_NODE_VOLUME_SIZE: "50"
+      OPENSHIFT_INFRA_NODE_VOLUME_TYPE: gp3
+      ZONES_COUNT: "3"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    test:
+    - ref: openshift-qe-ingress-perf
+    workflow: create-infra-move-ingress-monitoring-registry
```

Comment:
Also, update your test as below instead and give it a try.

    pre:
    - chain: ipi-aws-pre
    - chain: workers-scale
    - chain: create-infra-move-ingress-monitoring-registry
    workflow: openshift-qe-ingress-perf

Reply:
Done, ptal.